### PR TITLE
Adding the umbrella chart for amazon

### PR DESCRIPTION
**Pull request by chajain**

### CODE REVIEW

**Comment by mbelur** [comment]
Should we default to using the rancher version to 2.7.5 instead of 2.7.4 for now and then use the 2.7.6 as the changes to support payg offering will be in 2.7.6?

File: charts/umbrella/amazon/Chart.lock [hunk 1]
```diff
@@ -0,0 +1,15 @@
+dependencies:
+- name: rancher
+  repository: https://releases.rancher.com/server-charts/stable
+  version: 2.7.5
```
Comment (guangyee) [inline charts/umbrella/amazon/Chart.lock]: May want to make the imageTag to 2.7.5 to be consistent

File: charts/umbrella/amazon/values.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        
+  installCRDs: true
+  # The Fully Qualified Domain Name for your Rancher Server
+  hostname: "rancher.my.org"
+  global:
+    cattle:
+      psp:
+        enabled: false
+  # Set a bootstrap password. If leave empty, a random password will be generated.      
+  bootstrapPassword: "rancher"
```
Comment (guangyee) [inline charts/umbrella/amazon/values.yaml]: I think the bootstrapPassword is randomly generated if absent. Setting a default one will be problematic from security standpoint.

File: charts/umbrella/amazon/values.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  # Number of replicas of Rancher pods
+  replicas: 3
```
Comment (guangyee) [inline charts/umbrella/amazon/values.yaml]: shouldn't replicas be default to 1?

File: charts/csp-rancher-usage-operator/100.0.0+up0.1.0/values.yaml [hunk 4]
```diff
@@ -20,4 +20,4 @@ affinity: {}
 # readme/docs for more details
 additionalTrustedCAs: true
 rancherImage: rancher/rancher
-rancherImageTag: v2.7.3
+rancherImageTag: v2.7.4
```
Comment (guangyee) [inline charts/csp-rancher-usage-operator/100.0.0+up0.1.0/values.yaml]: v2.7.5?

**Review by guangyee — CHANGES_REQUESTED** [review]
Also, should we be using aws instead amazon? Though I don't have a strong opinion on this one.

**Comment by chajain** [comment]
Decide to go with deployer option

**Pull request closed without merging**